### PR TITLE
L2 polish

### PR DESCRIPTION
**Pull request by christianbaroni**

### CODE REVIEW

File: src/screens/WalletConnectApprovalSheet.js [hunk 1]
```diff
@@ -173,7 +173,10 @@ export default function WalletConnectApprovalSheet() {
     return {
       chainId: ethereumUtils.getChainIdFromNetwork(approvalNetwork),
       color: networkInfo[approvalNetwork]?.color,
-      name: capitalize(value?.charAt(0)) + value?.slice(1),
+      name:
```
Comment (brunobar79) [inline src/screens/WalletConnectApprovalSheet.js:176]: Better to use `networkInfo[approvalNetwork]?.name`

**Review by brunobar79 — APPROVED** [review]
🔥

**Review by skylarbarrera — APPROVED** [review]
lgtm

File: src/components/coin-icon/ChainBadge.js [hunk 2]
```diff
@@ -65,8 +91,14 @@ export default function ChainBadge({
       badgeXPosition={badgeXPosition}
       badgeYPosition={badgeYPosition}
       iconSize={iconSize}
+      marginBottom={marginBottom}
```
Comment (skylarbarrera) [inline src/components/coin-icon/ChainBadge.js:94]: this is null unless a value is passed and is causing a crash, not sure what you want the default margin to be

**Comment by skylarbarrera** [comment]
ran e2e locally, all good. issue is with WC

**Pull request merged**